resources/js/Components/LanguageSelector.vue

```diff
@@ -73,22 +74,22 @@ const shownLanguages = computed<Language[]>(() => {
 
 function onInput(searchedLanguage: string): void {
 	searchInput.value = searchedLanguage;
```

Comment:
If I write something and then remove it, `searchInput.value` is empty and the code makes an API call with an empty language code string.

Reply:
Not good! should work better now